### ISSUE TITLE
What is the replacement for deprecated import syntax?

### ISSUE THREAD

**Issue by copperlight** [openllb/hlb#223]
The managed artifacts guide recommends using the following:

```
import nodejs "./nodejs.hlb"
```

Which results in the following deprecation warning:

```
error: import path without keyword "from" is deprecated
runtime/build.hlb:1:15:
  │ 
1 │ import nodejs "./nodejs.hlb"
  │               ^^^^^^^^^^^^^^
  │               import path without keyword "from" is deprecated
[+] Building 21.2s (12/12) FINISHED 
```

I attempted to follow the `from` syntax introduced in the remote filesystem example, except pointing to a local file:

```
import nodejs from fs {
	local "./nodejs.hlb"
}
```

This throws a stack trace:

```
[+] Building 0.2s (1/1)                                                                                                                                                                              
 => [import nodejs] local://sha256:<redacted> (nodejs.hlb)                                                                                0.1s
 => => transferring sha256:<redacted>: 32B                                                                                                0.1s
strings: negative Repeat count
goroutine 1 [running]:
runtime/debug.Stack(0xc0006baef8, 0x2173e00, 0x26b0300)
	/usr/local/go/src/runtime/debug/stack.go:24 +0x9d
```

I suppose this violates the contract, but then I am not sure how local imports are supposed to work.

> The only contract is that the resulting fs must contain a module.hlb file.

**Comment by coryb** [comment]
If you import from an `fs` there must be a `module.hlb` file.  Otherwise you can import directly from a path:
```
import nodejs from `./nodejs.hlb`
```

**Comment by copperlight** [comment]
Switching to the following syntax:

```
import nodejs from "./nodejs.hlb"
```

produces the following error instead:

```
error: open runtime/runtime/nodejs.hlb: no such file or directory
runtime/build.hlb:1:20:
  │ 
1 │ import nodejs from "./nodejs.hlb"
  │                    ^^^^^^^^^^^^^^
  │                    no such file
```

With files arranged like so:

```
❯ tree
.
├── baseos
│   ├── build.hlb
│   └── module.hlb
├── candidate
├── manifest.json
└── runtime
    ├── build.hlb
    ├── module.hlb
    └── nodejs.hlb
```

**Comment by coryb** [comment]
things are always relative to the current document, so your `baseos/build.hlb` file would look something like:
```
import nodejs from "../runtime/nodejs.hlb"
```

**Comment by coryb** [comment]
Oh whoops, I see, you had that un the runtime/build.hlb, looks like a bug with double `./runtime/runtime`

**Comment by hinshun** [comment]
Also fixed the panic you saw earlier in: #225
